pgml-extension/sql/install/models.sql

```diff
@@ -200,9 +200,16 @@ CREATE OR REPLACE FUNCTION pgml.model_predict(
 	model_id BIGINT, -- `id` from `pgml.models`
 	features DOUBLE PRECISION[] -- list of features that the model accepts
 )
-RETURNS DOUBLE PRECISION
+RETURNS DOUBLE PRECISION[]
```

Comment:
This is the only real different with predict_multi. Why not use that instead?

Reply:
`predict_multi` is not model specific, this one is.